src/app/(proper_react)/redesign/(authenticated)/user/dashboard/View.tsx

```diff
@@ -137,24 +149,32 @@ export const View = (props: Props) => {
     props.userScanData.results,
     props.userBreaches
   );
-  const isAllFixed =
-    dataSummary.dataBreachFixedExposuresNum ===
-      dataSummary.dataBreachTotalExposuresNum &&
-    dataSummary.dataBrokerFixedExposuresNum ===
-      dataSummary.dataBrokerTotalExposuresNum;
+
+  const hasExposures = combinedArray.length > 0;
+  const hasUnresolvedBreaches =
+    filteredExposures.filter((exposure) => !isScanResult(exposure)).length > 0;
```

Comment:
Hmm... This might be a Product/Design, but do we actually want the numbers at the top to update if filters are active? And maybe more importantly: do we want the CTA and text above say something like "All breaches resolved" if we're just filtering out breaches?

Or am I misunderstanding what this does? (And the other lines below that base themselves on `filteredExposures` as well.)

Reply:
You understood that correctly! My intention was not to use the filtered exposures to determine the unresolved ones per tab. Fixed that in [7120b8e](https://github.com/mozilla/blurts-server/pull/3493/commits/7120b8e2338be0aacf565dec5748bfffda62da20).